mlflow/utils/databricks_utils.py

```diff
@@ -817,13 +917,18 @@ def get_databricks_runtime_major_minor_version():
     return DatabricksRuntimeVersion.parse()
 
 
-def _init_databricks_cli_config_provider(entry_point):
+_dynamic_token_config_provider = None
```

Comment:
why do we need this global variable?

Reply:
in databricks runtime notebook, we will initialize it (it reads token via dbutils APIs)